lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -207,6 +209,12 @@ def self.is_firebasecmd_supported?(cmd)
 
         true
       end
+      def self.post_notes(app_id, release_id, release_notes)
+       # payload = {releaseNotes: {releaseNotes: release_notes}}
+        connection.post("#{PATH}#{app_id}/releases/#{release_id}/notes", {releaseNotes: {releaseNotes: release_notes}}.to_json) do |request|
```

Comment:
Was not sure if we should inline it or not so I commented out the initial approach.

Reply:
Would be good to reduce the length of this line by setting up the payload and/or the URL above